user/tests/unit/string_interpolate.cpp

```diff
@@ -109,7 +109,8 @@ class Mocks {
                 String deviceID = spark_deviceID();
                 size_t id_len = deviceID.length();
 
-                SystemVersionInfo sys_ver;
+                SystemVersionInfo sys_ver = { sizeof(SystemVersionInfo), 0, 0, {0} };
```

Comment:
@mrlambchop I'd suggest against using this pattern in C++ and especially for structs that may very well be modified and grow and be explicit about what is initialized:

Either (preferred):
```c++
// Zero-initialize completely
SystemVersionInfo sys_ver = {};
sys_ver.size = sizeof(sys_ver);
```

or if the size field is the first member:

```c++
// Designated initializer for size, rest gets zero-initialized.
SystemVersionInfo sys_ver = {
    .size = sizeof(sys_ver)
};
```